common/Resources/ti.internal/extensions/binding.js

```diff
@@ -83,3 +83,8 @@ export function redirectCoreModuleIdToPath(coreModuleId, internalPath) {
 export function bindObjectToCoreModuleId(coreModuleId, object) {
 	bindings.set(coreModuleId, object);
 }
+
+global.redirectCoreModuleIdToPath = redirectCoreModuleIdToPath;
```

Comment:
So this is a useful method internally for me to hijack require to expose our shims via the "core" names used by Nodes.

I worry, because I never actually wrote this or intended it as publicly available API. This would re-expose it, but I never put any thought into making a special new API to do this - or whether we'd want to.

I would think we'd want to at least refactor it to some better API naming/usage if we do want to expose it.